### ISSUE TITLE
Need to restart webpack-dev-server to update ts check status after making change to my types in `d.ts` files?

### ISSUE THREAD

**Issue by cbdeveloper** [TypeStrong/fork-ts-checker-webpack-plugin#503]
## Current behavior

When I make changes to my types. Sometimes the plugin will only show/resolve TS errors/warnings after I stop and restart `webpack-dev-server`.

## Expected behavior

The plugin would update the TS errors/warning immediately after type changes.

## Steps to reproduce the issue

Don't know exactly how to reproduce, but would like to know if this is normal behavior.

## Issue reproduction repository

**For example**

The plugin is complaining about a `IS_MOBILE` property that is missing from my `LAYOUT` type.

![image](https://user-images.githubusercontent.com/43407798/91018642-bf6cb500-e5e7-11ea-8381-6ac9de1bc1af.png)

This is because I was experimenting a new shape for this object.

But you can see that **I already removed that property from my LAYOUT interface**, so that error should not be showing anymore.

![image](https://user-images.githubusercontent.com/43407798/91018700-d0b5c180-e5e7-11ea-965f-32cb476ad6f5.png)

Now I'm going to restart webpack:

![image](https://user-images.githubusercontent.com/43407798/91018721-d7dccf80-e5e7-11ea-8d8c-4be9315cfd3f.png)

You can see from the image above that the error is now gone.

**QUESTION**

Is this the normal behavior for the `ForkTsCheckerWebpackPlugin`? Do I need to restart my `webpack-dev-server` when I make those kind of changes? Or is it maybe cause because I'm using react-hot-loader or some other config I should be using. Has anyone seen this before?

**NOTE:** The normal VSCode ts check throughout my opened files is working fine. They always immediately reflect the changes I make to the types I'm using.

**PS:** Those types are declared in `d.ts` files using declare namespace {...}

## Environment

```
"fork-ts-checker-webpack-plugin": "^5.0.14",
"webpack": "^4.44.0",
```

**Comment by piotr-oles** [comment]
Duplicate: #36 . It's an old issue, I hope I will be able to solve it soon.

**Comment by cbdeveloper** [comment]
Thanks a lot for your reply. Sorry about the duplicate. This package helps me a lot!
